plaidml2/core/ffi.cc

```diff
@@ -53,15 +63,31 @@ void plaidml_init(plaidml_error* err) {
       }
       IVLOG(1, "plaidml_init");
       Settings::Instance()->load();
+      auto ctx = GlobalContext::getContext();
       GetPlatform();
+      auto eventlog_str = vertexai::env::Get("PLAIDML_EVENTLOG_FILENAME");
+      if (eventlog_str.size()) {
+        IVLOG(1, "Logging events to " << eventlog_str);
+        vertexai::eventing::file::proto::EventLog e_config;
+        e_config.set_filename(eventlog_str);
+        auto eventlog = std::make_shared<vertexai::eventing::file::EventLog>(e_config);
+        ctx->set_eventlog(std::move(eventlog));
+        ctx->set_is_logging_events(true);
+      }
     });
   });
 }
 
+void plaidml_reset_eventlog(void) {
```

Comment:
Also, let's throw this in the default namespace so that it's not visible to other translation units.